config/initializers/prometheus_exporter.rb

```diff
@@ -2,7 +2,7 @@
 
 # check for environment variable so that we can run production rake task in Dockerfile
 # otherwise we try to load the metrics outside of the actual app itself
-if ENV['PROMETHEUS_METRICS'].in? ['on', 'true']
+if ['on', 'true'].include? ENV['PROMETHEUS_METRICS']
```

Comment:
sure these downcase? :) 
does rubocop like this or it prefers the `w%[on true]`

Reply:
rubocop seems happy 😄